### ISSUE TITLE
Genie Builder with GenieFramework.jl?

### ISSUE THREAD

**Issue by rohitsathish** [GenieFramework/GenieFramework.jl#6]
Is there a way to use the Genie Builder extension on VSCode with the new GenieFramework.jl API?

**Comment by essenciary** [comment]
Right now you can use the insider version of the plugin (just look for it in the marketplace). And soon (maybe even today) it will also be available as a new release of the stable plugin.

**Comment by rohitsathish** [comment]
Thanks. Any tutorials/info on how the two would work together?

Kind Regards,
Rohit Sathish

**Comment by essenciary** [comment]
Yes, there will be a video tutorial and some demos to start with.
